qaframework-bdd-tests/src/test/resources/features/refapp-2.x/17-Service/service.feature

```diff
@@ -0,0 +1,22 @@
+Feature:Service Management
+
+  Background:
+    Given user goes to the appointment scheduling app
+    When user goes to manage service page
+    Then user clicks on new service page
+    And user enters service details
+
+  @selenium
+  @service
+  Scenario: Service management
+    And user clicks on save button
+    And system checks the saved service
+    #edit section
+    And user clicks on edit service type icon
+    And user clicks again on save button
+    And system checks the editted service type
+    #delete section
+    And user clicks on delete service type icon
+    And user clicks on comfirm button
+    Then user checks the availability of service type by name
+
```

Comment:
Isn't one empty line enough to cater for git collaborators?

Reply:
Resolved